lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -41,10 +41,11 @@
             EmailFieldView: FieldViews.TextFieldView.extend({
                 fieldTemplate: field_text_account_template,
                 successMessage: function () {
-                    return this.indicators.success + StringUtils.interpolate(
+                    return HtmlUtils.joinHtml(
+                        this.indicators.success, StringUtils.interpolate(
                             gettext('We\'ve sent a confirmation message to {new_email_address}. Click the link in the message to update your email address.'), /* jshint ignore:line */
                             {'new_email_address': this.fieldValue()}
-                        );
+                        ));
```

Comment:
These two `)` should be split on to their own lines, indented the same as the first line that opens the block.